packages/edit-widgets/src/index.js

```diff
@@ -71,6 +72,13 @@ export function initialize( id, settings ) {
 		);
 	} );
 
+	setFeatureDefaults( 'core/edit-widgets', {
+		fixedToolbar: false,
```

Comment:
Do you think that `fixedToolbar` should be set per screen or should it be a global setting?

It's out of scope for this PR, but it's something that might become a point of frustration when there are screens for posts, widgets, navigation, and the site. On the other hand, it might be difficult to find an easy way to satisfy all use cases. Some folks might want to have this setting different per screen, but others might want to have this change to be applied globally 😅

Reply:
Yep, hard to say. I discussed it a bit with @tellthemachines in a comment above (second comment under her review - https://github.com/WordPress/gutenberg/pull/33774#discussion_r686502494). There are some options available now, we just have to pick a strategy and implement it 😄

Reply:
Cool, it looks like we are on the same page. We don't know what's the best way forward but at least we are aware that the issue exist 👍🏻